doc/nrf/libraries/bluetooth_services/mesh/light_ctrl_cli.rst

```diff
@@ -7,6 +7,11 @@ Light Lightness Control Client
    :local:
    :depth: 2
 
+.. note::
+   This model interacts with the Sensor API, which was rewritten in |NCS| v2.6.0, and this is what is documented in this release.
+   The old API is deprecated, but is still available by enabling :kconfig:option:`CONFIG_BT_MESH_SENSOR_USE_LEGACY_SENSOR_VALUE`, which is enabled by default in the deprecation period.
+   See the documentation for |NCS| v2.5.2 for documentation about the old Sensor API.
+
```

Comment:
```suggestion
.. note::
   This model interacts with the new sensor API introduced as of |NCS| v2.6.0.
   The old API is deprecated, but still available by enabling the Kconfig option :kconfig:option:`CONFIG_BT_MESH_SENSOR_USE_LEGACY_SENSOR_VALUE`.
   The Kconfig option is enabled by default in the deprecation period.
   See the documentation for |NCS| versions prior to v2.6.0 for documentation about the old sensor API.

```

Reply:
And the same for the other occurrences :)

Reply:
What to do in the other cases? The opening line is different